gui/system/views.py

```diff
@@ -1009,8 +1010,10 @@ def debug_download(request):
     gc = GlobalConfiguration.objects.all().order_by('-id')[0]
 
     _n = notifier()
-    if not _n.is_freenas() and _n.failover_licensed():
-        debug_file = '%s/debug.tar' % direc
+    dual_node_debug_file = debug_file = '{}/debug.tar'.format(direc)
+
+    if not _n.is_freenas() and _n.failover_licensed() and os.path.exists(dual_node_debug_file):
+        debug_file = dual_node_debug_file
```

Comment:
Actually I've called it "dual_node_debug_file" to be more verbose and and make the code easier to understand - then the if statement gives explicit suggestion what types of debug dump you can expect.

We can make it either way. What's your OCD suggestion Suraj ? :D